build-config.js

```diff
@@ -93,11 +93,7 @@ exports.options = {
     keyFilename: __dirname + '/gcskeyfile.json', // Your keyFilename path
     bucket: '${gcsBucket}',
   },
-  'preview config': {
-    protocol: 'http',
-    host: 'twreporter-react', // Using host entries created by Docker in /etc/hosts
-    port: 3000
-  },
+  'preview origin': 'http://localhost:3000',
```

Comment:
I think the naming should be `preview host` rather than `preview origin` as the keystone sends the request for `preview` resource to the `preview host server`.  `Origin` tends to describe where the request is from (i.e. keystone)

Reply:
Refer to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin , 
I think `preview origin` is kind of accurate.

Reply:
I think the point is that the context is different. As for your reference, the "origin" of the URL is `https://mozilla.org`.  However, we are trying to configure the "address of the server" which is used in turn to build the preview article URL. From the perspective of the final URL, the server serves as the "origin" but when it comes to describe the server providing preview server, it would be sort of weird and confusing.  That's my two cents. I don't want to take this too far so you can move forward if you still think the naming is appropriate.